library/Makefile

```diff
@@ -91,9 +104,7 @@ shared: libmbedcrypto.$(DLEXT) libmbedx509.$(DLEXT) libmbedtls.$(DLEXT)
 # tls
 libmbedtls.a: $(OBJS_TLS)
 	echo "  AR    $@"
-	$(AR) -rc $@ $(OBJS_TLS)
-	echo "  RL    $@"
-	$(AR) -s $@
+	$(AR) $@ $(OBJS_TLS)
```

Comment:
I have not removed the options. They are added to ```$(AR)``` variable at line https://github.com/mazimkhan/mbedtls/blob/e6a6f0b3713ae3ed32caf57ef0744b3ad11c227f/library/Makefile#L17
Hence similar to your suggestion.